### PR TITLE
implement momo payment qr-code page

### PR DESCRIPTION
**Pull request by dinhhieu240901**
Everyone, please help me review and approve this PR

### CODE REVIEW

File: src/main/webapp/img/logo_momo.png
Comment (vietvu1996) [inline src/main/webapp/img/logo_momo.png:1]: Nice picture

**Review by thaonguyen011 — APPROVED** [review]
great

File: src/main/webapp/signup/payment.jsp
Comment (vietvu1996) [inline src/main/webapp/signup/payment.jsp:1]: No conflict in this code. Confirmed

**Review by vietvu1996 — APPROVED** [review]
Checked

**Pull request merged**